crates/apollo-smith/src/name.rs

```diff
@@ -90,7 +90,7 @@ impl<'a> DocumentBuilder<'a> {
     // Mirror what happens in `Arbitrary for String`, but do so with a clamped size.
     pub(crate) fn limited_string(&mut self, max_size: usize) -> ArbitraryResult<String> {
         loop {
-            let size = self.u.int_in_range(0..=max_size)?;
+            let size = self.u.int_in_range(1..=max_size)?;
```

Comment:
An empty result string is rejected below and causes this loop to start over, so let’s ask for a non-zero size